docs/COMPILE.md

```diff
@@ -43,6 +43,7 @@ sudo systemctl restart systemd-binfmt
   _For Pi4. Change to RPI2 or RPI3 for other models.  Change `-DRPI4=1` to `-DRPI4ARM64=1` for compiling on arm64. (armhf multiarch or chroot required alongside armhf gcc. Install it with 'sudo apt install gcc-arm-linux-gnueabihf'.)_
 
 ```
+sudo apt install gcc-arm-linux-gnueabihf # optional: armhf cross-compiler only required if you are not in a multiarch or chroot enviroment
```

Comment:
the `RPI4ARM64` profile makes explicit use on `gcc-arm-linux-gnueabihf`. So it's aboslutly needed even on multiarch environment or chroot environement.

I'm not sure I understand the `# optionnal:....` comment here.

Reply:
ooh. I misunderstood the purpose of the target. I will fix this tomorrow